### PR TITLE
add go example

### PR DESCRIPTION
**Pull request by shunsuke-h-opn**
#### 1. Objective
I added golang example for Omise.

#### 2. Description of change
These are what I added.

golang(PayPay)

#### 3. Impact of the change
None.

#### 4. Priority of change
Normal.

### CODE REVIEW

File: go/.env.example [hunk 1]
```diff
@@ -0,0 +1,10 @@
+DB_USER=root
+DB_PASS=
+DB_NAME=go_db
+DB_HOST=localhost
+DB_PORT=3306
+OMISE_PUB_KEY=pkey_test_sssss
```
Comment (danfowler) [inline go/.env.example:6]: This doesn't match the README. Nor does it match the other examples.

File: go/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+It has some examples.
+- Credit Card Payment
+- PayPay
+
+## Requirements
+To follow along, you will need:
+
+An Omise account (and your $OMISE_PUBLIC_KEY and $OMISE_SECRET_KEY)
+
+- golang 1.16
```
Comment (danfowler) [inline go/README.md:14]: This version is EOL as of March 2022.

File: go/go.mod [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	github.com/gin-contrib/gzip v0.0.1
+	github.com/gin-gonic/gin v1.7.2
+	github.com/go-gorp/gorp v2.2.0+incompatible
+	github.com/go-playground/validator/v10 v10.8.0
+	github.com/go-sql-driver/mysql v1.7.1
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/google/uuid v1.3.0
+	github.com/joho/godotenv v1.3.0
+	github.com/json-iterator/go v1.1.11 // indirect
+	github.com/lib/pq v1.5.2 // indirect
```
Comment (danfowler) [inline go/go.mod]: Why do we have a postgres driver here?

File: go/.env.example (hunk 1 above)
Comment (shunsuke-h-opn) [reply go/.env.example:6]: sure, I will check

File: go/go.mod (hunk 3 above)
Comment (shunsuke-h-opn) [reply go/go.mod]: Sorry, it's my mistake.
I will remove it.

File: go/README.md (hunk 2 above)
Comment (shunsuke-h-opn) [reply go/README.md:14]: @danfowler 
Updated to 1.21
https://github.com/omise/examples/pull/32/commits/d846de0b3d8fed816b983ac34a8419b95ba6a282

File: go/go.mod (hunk 3 above)
Comment (shunsuke-h-opn) [reply go/go.mod]: @danfowler 
remove it.
https://github.com/omise/examples/pull/32/commits/d2f4b3d3f976cf2c3561adbfab53bd4ae02918eb

File: go/.env.example (hunk 1 above)
Comment (shunsuke-h-opn) [reply go/.env.example:6]: > This doesn't match the README.

I can not find that does not match the README.

it looks same,

Comment (shunsuke-h-opn) [reply go/.env.example:6]: > This doesn't match the README.

I can not find that does not match the README.

it looks same.

Screenshot 2023-11-14 at 14 18 31

**Review by danfowler — APPROVED** [review]

**Review by danfowler — APPROVED** [review]

**Pull request merged**